internal/olm/operator/internal/registry_pod_test.go

```diff
@@ -0,0 +1,29 @@
+package olm
+
+import (
+	"testing"
+
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func newFakeClient() kubernetes.Interface {
+	fakekubeclient := fake.NewSimpleClientset()
+
+	return fakekubeclient
+}
+
+// TODO: Add unit tests
+func TestCreateBundleRegistry(t *testing.T) {
```

Comment:
Can you make these ginkgo/gomega tests?

Reply:
@estroz did we already switch to ginkgo/gomega?

Reply:
ginkgo/gomega tests will be done in a follow-up PR